security-center/snippets/v2/updateMuteRule.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * Updates an existing mute configuration.
+ */
+function main(organizationId, muteConfigId, location = 'global') {
+  // [START securitycenter_update_mute_config_v2]
+  // Imports the Google Cloud client library.
+  const {SecurityCenterClient} = require('@google-cloud/security-center').v2;
+
+  // Create a Security Center client
+  const client = new SecurityCenterClient();
+
+  const name = `organizations/${organizationId}/locations/${location}/muteConfigs/${muteConfigId}`;
+  /**
+   *  The list of fields to be updated.
+   *  If empty all mutable fields will be updated.
+   */
+  const updateMask = {
+    paths: ['description'],
+  };
```

Comment:
discussion: Our samples so rarely showcase how to use an updateMask that it's kind of nice to see.

question: Will all samples showing update use the updateMask? It should be all or none.

Reply:
I am not sure about other samples though, may be it's good to have it.